products/idn/docs/identity-now/saas-connectivity/connector-commands/source-data-read.md

```diff
@@ -0,0 +1,116 @@
+---
+id: source-data-read
+title: Source Data Read
+pagination_label: Source Data Read
+sidebar_label: Source Data Read
+keywords: ['connectivity', 'connectors', 'Source Data Read']
+description: Create account on the source.
+slug: /docs/saas-connectivity/commands/source-data-read
+tags: ['Connectivity', 'Connector Command']
+---
+
+| Input/Output |            Data Type        |
+| :----------- | :-------------------------: |
+| Input        | StdSourceDataDiscoverInput  |
+| Output       | StdSourceDataDiscoverOutput |
```

Comment:
did you mean sourcedataread input/output here?

Reply:
Yes, just pushed a fix for this, thanks for reading!